airbyte-integrations/connectors/source-asana/metadata.yaml

```diff
@@ -1,32 +1,46 @@
 data:
-  ab_internal:
-    ql: 300
-    sl: 100
   allowedHosts:
     hosts:
       - app.asana.com
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
+  releases:
+    breakingChanges:
+      1.0.0:
+        upgradeDeadline: "2024-04-15"
+        message: 
+          - >-
+            "The verison migrates the TrustPilot connector to the low-code framework for greater maintainability.
```

Comment:
this is referring to trustpilot. I don't think a breaking change is required here so you should be able to remove the whole block